apps/meteor/app/api/server/v1/rooms.ts

```diff
@@ -962,3 +964,37 @@ API.v1.addRoute(
 		},
 	},
 );
+
+API.v1.addRoute(
+	'rooms.close',
```

Comment:
wonder why this is called `.close` if we call `hideRoom` inside 🤔

Reply:
I wondered the same thing on the FE. Client-side it is "hide room", server side "close room" (`channels.close`, `dm.close`), probably as a counterpart to the "open room" (`channels.open`, `dm.open`).

Considering this is an old name convention, do you think it is worth it renaming it to `rooms.hide` and align FE and BE?

Reply:
Going with `rooms.hide` to keep the endpoint name true to its function.